app/views/shared/_error_messages.html.erb

```diff
@@ -0,0 +1,12 @@
+<% if @user.errors.any? %>
+  <div id="error_explanation">
+    <div class="alert alert-danger">
+      The form contains <%= pluralize(@user.errors.count, "error") %>.
```

Comment:
Nghiên cứu các thể hiên số nhiều với I18n để sử udngj chỗ này em nhé https://viblo.asia/p/i18n-in-rails-part-2-GrLZDAO3lk0